tools/wptserve/wptserve/config.py

```diff
@@ -186,20 +223,28 @@ def __init__(self,
                  subdomains=set(),
                  not_subdomains=set(),
                  config_cls=None,
+                 proxy_logger=None,
                  **kwargs):
 
         self._data = self._default.copy()
         self._ssl_env = None
 
         self._config_cls = config_cls or self.default_config_cls
 
-        if logger is None:
-            self._logger_name = "web-platform-tests"
+        if proxy_logger is not None and logger is not None:
+            raise ValueError("Can't provide both proxy_logger and logger")
+
+        if proxy_logger is not None:
+            self._proxy_logger = proxy_logger
         else:
-            level_name = logging.getLevelName(logger.level)
-            if level_name != "NOTSET":
-                self.log_level = level_name
-            self._logger_name = logger.name
+            self._proxy_logger = None
+            if logger is None:
+                self._logger_name = "web-platform-tests"
```

Comment:
Does this need to be unique? If not, maybe "wptserve" gives a better clue? Or if it doesn't appear anywhere at all, just the empty string :)